packages/docusaurus-init/templates/classic-typescript/tsconfig.json

```diff
@@ -1,5 +1,10 @@
 {
   // This file is not used in compilation. It is here just for a nice editor experience.
   "extends": "@tsconfig/docusaurus/tsconfig.json",
+  "compilerOptions": {
+    "paths": {
+      "@site/*": ["./*"]
+    }
+  },
```

Comment:
hmm, looks like another thing we could add to the base config 🤪 

Opened a PR here: https://github.com/tsconfig/bases/pull/71

Reply:
Sure, let's hold on till that's merged